src/security-testing/static-analysis.md

```diff
@@ -0,0 +1,55 @@
+---
+tags:
+  - Engineer/Developer
+  - Security Specialist
+  - Operations & Strategy
+  - Devops
+  - SRE
+contributors:
+  - role: wrote
+    users: [patrickalphac]
+---
+
+# Static Analysis
+
+Static analysis is when we look at our code or have some tool look at our code without actually executing it. Unlike dynamic testing (unit tests, fuzz tests, integration tests), static analysis examines your code's structure, syntax, and patterns to identify potential vulnerabilities and code quality issues.
```

Comment:
That's a definition tailored a small subset of static analysis, which is to use static analysis for bug findings :) 

Not sure of how far we want to go here, but I wrote a tldr intro on static analysis here: https://secure-contracts.com/program-analysis/slither/docs/src/api/static_analysis.html

If we don't want to go too much into the details, maybe we can say
```
At a high level, static analysis examines the structure, syntax, and patterns of your code without executing it. There are many forms of static analysis, and compilers like solc rely on these techniques. From a security perspective, static analysis can help identify potential vulnerabilities and code quality issues.
```

Reply:
Great!